api/definitions/v1.buildoutput/description.adoc

```diff
@@ -0,0 +1 @@
+A BuildOutput is input to a build strategy and describes the Docker image that the strategy should produce. The BuildOutput defines an optional location to push the output of this build to. This value will be used to look up a Docker image repository to push to. The output types supported are <<v1.ImageStreamTag>> and `DockerImage`. In the case of an `ImageStreamTag`. the `ImageStreamTag` will be looked for in the namespace of the build unless another namespace is specified. The BuildOutput contains a <<v1.PushSecret>>that is the name of the <<v1.Secret>> that should be used for setting up the authentication for executing the Docker push to an authentication-enabled Docker Registry (or Docker Hub).
```

Comment:
> A BuildOutput is input to a build strategy and describes the Docker image that the strategy should produce. The BuildOutput defines an optional location to push the output of this build to. This value will be used to look up a Docker image repository to push to. The output types supported ... 

A BuildOutput (optional) specifies the output location of a build. It contains a reference that can be resolved to a  Docker repository where the resulting build image should be pushed. The supported output reference types ... 

> The output types supported are <<v1.ImageStreamTag>> and `DockerImage`. 

The output types supported are <<v1.ImageStreamTag>> and `DockerImage`. If `DockerImage` is specified, the Name of the reference should contain a docker repository spec.